Puppetfile

```diff
@@ -2,6 +2,11 @@
 
 mod 'stahnma/epel', '1.2.2'
 
+# Modules that have been extracted from core Puppet
+mod 'puppetlabs/augeas_core', '1.0.0'
+mod 'puppetlabs/sshkeys_core', '1.0.0'
+mod 'puppetlabs/yumrepo_core', '1.0.0'
+
```

Comment:
Does r10k support semantic versions like '~> 1.0'?

Reply:
It doesn't look like it [from the documentation](https://github.com/puppetlabs/r10k/blob/master/doc/puppetfile.mkd). The only options it lists are:
* Getting the latest, because you didn't specify.
* Getting a specific version.
* Specifying you always want the latest version.